### DISCUSSION TITLE
Expand all operations by default in the classic layout

### DISCUSSION THREAD

**Discussion by crukundo** [scalar/scalar#2128] [General]
As it is now, any tag that has more than 1 operation will have a definition, a card listing all the endpoints and a "Show More" button that expands when clicked to show the nested operations. Is there a possibility to programatically expand all the sections under each tag? Basically remove the Show More button and have all sections expanded to reveal all endpoints by default. What do you think?

<img width="1117" alt="Screenshot 2024-06-17 at 17 47 49" src="https://github.com/scalar/scalar/assets/7932662/08a5ba03-0b33-4151-be1a-9cd34d609340">

**Comment by hanspagel** [comment]
Love it! We do this for performance reasons, but after some recent optimisations we might be able to further increase the limit of when we hide operations.

**Reply by crukundo** [reply]
Awesome. Looking forward.

**Comment by richardi-gentrack** [comment]
`any tag that has more than 1 operation` - for consistency, it would be nice for ALL tags (including those with only one operation) to be folded with `Show More`
